src/test/java/net/snowflake/client/jdbc/SnowflakeColumnMetadataTest.java

```diff
@@ -0,0 +1,40 @@
+package net.snowflake.client.jdbc;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+import org.junit.Test;
+
+public class SnowflakeColumnMetadataTest {
+
+  @Test
+  public void testSnowflakeColumnMetaData() {
+    SnowflakeColumnMetadata metadata =
+        new SnowflakeColumnMetadata(
+            "mock", 1, true, 2, 3, 4, "fake", true, null, null, null, null, true);
+    List<FieldMetadata> list = mock(List.class);
+    metadata.setName("test");
+    metadata.setType(10);
+    metadata.setNullable(false);
+    metadata.setLength(12);
+    metadata.setPrecision(13);
+    metadata.setScale(14);
+    metadata.setTypeName("type");
+    metadata.setFixed(false);
+    metadata.setFields(list);
+    metadata.setAutoIncrement(false);
+
+    assertEquals(metadata.getName(), "test");
```

Comment:
let's not test getters and setters - let's test them indirectly

Reply:
The same as above. I removed the file.